rules/fixtures/rules.yaml

```diff
@@ -0,0 +1,5 @@
+groups:
+  - name: test
+    rules:
+    - record: job:http_requests:sum
+      expr: sum(http_requests) by (job)
```

Comment:
Let's be a little more realistic:  job:http_requests:rate5m = sum by (job)(rate(http_requests_total[5m]))